lib/v2/nju/radar.js

```diff
@@ -17,6 +17,14 @@ module.exports = {
                 target: '/nju/dafls',
             },
         ],
+        exchangesys: [
+            {
+                title: '本科生交换生系统',
+                docs: 'https://docs.rsshub.app/university.html#nan-jing-da-xue-ben-ke-sheng-yuan-jiao-huan-sheng-xi-tong',
+                source: ['/exchangesystem', '/'],
+                target: '/nju/exchangesys',
```

Comment:
`/nju/exchangesys` will not match any routes as there isn't any default value for `type`